apps/system-e2e/entrypoint.sh

```diff
@@ -3,5 +3,5 @@
 set -euo pipefail
 
 echo "Current test environment: ${TEST_ENVIRONMENT}"
-
-cypress run "$@"
+echo "Cypress args: $*"
+cypress run --browser chrome "$@"
```

Comment:
```suggestion
${TEST_BROWSER-chrome}
cypress run --browser $TEST_BROWSER "$@"
```

Reply:
we are experiencing timeouts still when using Chrome for some reason. We are thinking about removing the `--browser` flag and revert back to `electron` for now.

The flag will be set explicitly in the pipeline.